lib/lambcast_web/components/layouts/root.html.heex

```diff
@@ -4,9 +4,10 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="csrf-token" content={get_csrf_token()} />
-    <.live_title suffix=" · Phoenix Framework">
+    <.live_title>
       <%= assigns[:page_title] || "Lambcast" %>
     </.live_title>
+    <link rel="icon" type="image/x-icon" href="/images/lambcast.jpeg" />
```

Comment:
JPEG it's not the best image format for the favicon. Maybe we could change it later if we get a brand kit